### PR TITLE
[wallet-ext] show only 2 ledger accounts in the import flow

### PR DESCRIPTION
**Pull request by williamrobertson13**
## Description 

https://mysten-labs.slack.com/archives/C04R8B45QDS/p1681502281725109?thread_ts=1681493666.781589&cid=C04R8B45QDS

TLDR; Ledger wants us to make users confirm every request to `getPublicKey`, so we're only going to show 2 accounts until we work out some improvements to the Sui Ledger app and figure out the best pagination / load more / infinite scrolling UX.

## Test Plan 
- Manual testing

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)
- [X] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### Release notes
N/A

### CODE REVIEW

**Review by plam-ml — APPROVED** [review]

**Comment by williamrobertson13** [comment]
Actually nevermind on this, we don't need to make any urgent UX changes after all
